test/apps/iodemo/io_demo.cc

```diff
@@ -1662,6 +1670,16 @@ class DemoClient : public P2pDemoCommon {
         ++_num_completed;
         ++server_info.num_completed[op];
 
+        if (opts().per_conn_info && (opts().window_size == 1)) {
+            double elapsed = get_time() - server_info.ts_sent;
```

Comment:
maybe we should calculate it per operation to avoid `window_size == 1` limitation?

Reply:
w/ window_size == 1, we only need to store one timestamp per connection otherwise we need one timestamp per operation. w/ window_size > 1, we are seeking high bandwidth, calculating per operation might harm performance.

Reply:
but `window_size == 1` means that it's done per operation since traffic is limited by 1 operation in window. @yosefe  WDYT?
if the metric is interesting only for win == 1, then need to document that the `-V` parameter is ignored when  window_size != 1

Reply:
`-V` is not mutex with `window_size != 1`. `-V` enables printing additional per connection statistics while currently only aggregate statistics get printed. currently `window_size == 1` enables printing latency otherwise only throughtput get printed. the PR is not changing that.

Reply:
ok

Reply:
1. Doing get_time() for each operation is too expensive. do we really need to check latency per operation?
2. IMO checking window_size==1 is meaningless. Even with multiple connections, latency can grow because one connection's messages are queued after others. user should treat the report of latency value according to the number of connections and outstanding requests.